detection-rules/impersonation_expensify.yml

```diff
@@ -0,0 +1,46 @@
+name: "Brand impersonation: Expensify"
+description: "Impersonation of the expense management software Expensify."
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and (
+    strings.ilike(sender.display_name, '*expensify*')
+    or strings.ilike(sender.email.email, '*expensify*')
+    or strings.ilevenshtein(sender.email.domain.sld, 'expensify') <= 1
+  )
+  // negating listservs
```

Comment:
how does this impact messages that land in a google group?

Reply:
This wouldn't flag on messages delivered via google group, IIRC we've seen issues in the past and are waiting for better handling of groups in the platform.